source/components/composing-components.md

```diff
@@ -1,8 +1,11 @@
-Components really shine when you use them to their full potential, that is, by composing them with one another.
-Take for example the `<ul>` element, and the fact that only `<li>` elements are appropriate as children.
-If we want the same type of behavior, then we have to compose our components.
+Components organize applications best when leveraged with composition.
+The phrase "data-down", which is used often when discussing Ember's
+rendering layer, means to treat a user interface as if it were built from
+pure functions. By pure function, we mean a function that given the same input
+will always result in the same behavior.
```

Comment:
In #556, we had discussed making this less React-ish/jargony, along the lines of:

"Components organize applications best when the same input will always result in the same behavior. This approach is often called "data down, actions up" - components receive data and emit actions, but themselves never change."